rcjaRegistration/regions/utils.py

```diff
@@ -0,0 +1,28 @@
+from django.db.models import Q
+
+from regions.models import State, Region
+
+def getRegionsLookup():
+    """
+    Creates a list of objects with state id and a list of regions that are available for that state.
+    """
+    def createLookupObj(stateID, regions):
+        class RegionLookupObj:
+            pass
+
+        regionLookup = RegionLookupObj()
```

Comment:
What's the rationale in creating the RegionLookupObj, then immediately setting properties on it, as opposed to going through a constructor?

Reply:
Actually, what's the point of the createLookupObj function at all? This looks like it's essentially acting as a constructor for LookupObjs, with the notable exception that every single LookupObj will be of a differing type (due to the local instantiation of the class)

Reply:
Yeah good point, will change that